test/testcafe/spec/ChallengeOktaVerifyFastPassView_spec.js

```diff
@@ -86,7 +83,7 @@ const universalLinkMock = RequestMock()
   .onRequestTo(mockHttpCustomUri)
   .respond('<html><h1>open universal link</h1></html>');
 
-fixture('Device Challenge Polling View for user verification with the Loopback Server, Custom URI and Universal Link approaches');
+fixture('Device Challenge Polling View for user verification and MFA with the Loopback Server, Custom URI and Universal Link approaches');
```

Comment:
I will suggest to also cover the tests here https://github.com/okta/okta-signin-widget/pull/1554/files#diff-698534168a87ff567c25b350509b7261a3660769a3c5fd36b10adc3873731f39R212 this is closer to real scenario i think

Reply:
will not do this based on discussion